client-js/app/scripts/registry.ts

```diff
@@ -9,182 +11,10 @@ export interface RegistryListing {
   vals: Array<string>;
 }
 
-export interface RegistryApi {
```

Comment:
I don't really understand the distinction. And both seem to contain the same information, the command, the type to send and the expected type to return. Having what looked like the same information in two places in the same file, made the code harder for me to read (since I guess I didn't understand I was deleting something that had a purpose)